### ISSUE TITLE
Providing free Vercel hosting services to interested OpenJSF projects

### ISSUE THREAD

**Issue by ovflowd** [openjs-foundation/cross-project-council#1022]
As discussed in previous meetings within the Cross Project Council, we are considering a partnership with Vercel to allow us to use their services.

The talks so far have succeeded, and I'm happy to inform you that the Node.js TSC had no objections to the usage of Vercel for future projects, such as the Website (nodejs.org) transition to their platform and other side-projects, such as downloads CDN on their side, PR previews for the Node.js API docs, and others.

I'm creating this issue as we're having talks internally within the CPC, and a subset of people, that are helping with the coordination of the implementation of Vercel (What the partnership would be about, what we would get, what Vercel possibly want out of us, who would should the credentials, and all other sorts of topics).

#### The current team handling the partnership

- @Trott
- @joesepi 
- @rginn 
- @richardlau
- @kyliewd 
- @ovflowd 

### Current Task-list

- [x] Initial considerations and possible blockers
- [x] Forward the idea to the Node.js TSC
- [x] Get a ✅ from the Node.js TSC
- [x] Verify with the Foundation the next steps
  - Such as figuring out any other possible blocker
  - Verify what the partnership entails and what are the requirements from Vercel
- [x] Decide on the governance model
  - Who has access to the accounts, and how the permission model works
  - Document about the credentials/permission model
- [x] Document about the resource on Project Resources (resources that our project members can use)
- [x] Final remarks/Close the Issue

**Comment by ovflowd** [comment]
cc @leerob so you can keep track of the conversation here :)

**Comment by leerob** [comment]
Congrats on the new site being merged! Are we still on track here? How can I help?

**Comment by ljharb** [comment]
This issue title is a bit confusing; you're talking about using vercel for node's website, right?

**Comment by ovflowd** [comment]
@ljharb afaik we were talking about something were OpenJS would have a partnership with Vercel so any member project could use Vercel?

**Comment by ljharb** [comment]
ahhh gotcha, thanks for clarifying.

**Comment by ovflowd** [comment]
Sure :)

**Comment by tobie** [comment]
Can I suggest reframing this (in the title) as something along the lines of "Providing free Vercel hosting services to interested OpenJSF projects?"

The current issue title sounds very prescriptive (i.e. suggesting that projects would need to move away from their current solution to using Vercel). Of course this is not the case, but it's not clear from the topic.

**Comment by ovflowd** [comment]
@tobie, I think there's confusion here. This issue concerns the process of OpenJS closing a partnership with Vercel rather than projects needing to adopt Vercel. I believe the title fits its description. Otherwise, it would be written as "Proceed with the adoption of Vercel for the OpenJS Foundation member projects"...

But if it was confusing enough for you, let me rephrase it to your suggestion :)

(English is not my native language, so I honestly didn't see anything wrong with the current title)

**Comment by ovflowd** [comment]
Everything went well, but @bensternthal, should we document how member projects might request access to our Vercel account? Should this be part of the Infrastructure Menu?

**Comment by bensternthal** [comment]
Looks like we aren't quite ready to roll out Vercel broadly. We will wait until after the launches of OpenJS and Node on their platform.

**Comment by ovflowd** [comment]
That makes sense!

**Comment by tobie** [comment]
What's  the status, here? Can we close this issue or spell our next steps?

**Comment by ovflowd** [comment]
> What's the status, here? Can we close this issue or spell our next steps?

Afaik the issue should be closed. We already formed our partnership with Vercel.
